web-components/src/utils/schemaURL.ts

```diff
@@ -1,12 +1,28 @@
-export interface urlType {
+export interface UrlType {
   '@type': 'schema:URL';
-  '@value'?: string;
+  '@value'?: string | null;
 }
 
-export function getURLInitialValue(value?: urlType): urlType {
+export interface UrlList {
+  '@list': Array<UrlType>;
+}
+
+export function getURLInitialValue(value?: UrlType): UrlType {
   return (
     value || {
       '@type': 'schema:URL',
+      '@value': null,
+    }
+  );
+}
+
+export function getURLListInitialValue(
+  arrSize: number,
+  value?: UrlList,
+): UrlList {
+  return (
+    value || {
+      '@list': Array(arrSize).fill(getURLInitialValue(undefined)),
```

Comment:
This could also be written like this:

```suggestion
      '@list': Array.from({length: arrSize}, getURLInitialValue (undefined)),
```

Reply:
I didn't know this!